wrapper/src/test/java/integration/container/tests/AdvancedPerformanceTest.java

```diff
@@ -18,6 +18,9 @@
 
 import static org.junit.jupiter.api.Assertions.assertTrue;
 import static org.junit.jupiter.api.Assertions.fail;
+import static software.amazon.jdbc.PropertyDefinition.PLUGINS;
+import static software.amazon.jdbc.plugin.efm.HostMonitoringConnectionPlugin.*;
```

Comment:
Looks like there are some Checkstyle failures related to using wildcards in the import.